packaging/centos-7/build-image/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM alanfranz/fwd-centos-7:latest
```

Comment:
Why aren't you using the official [CentOS](https://hub.docker.com/_/centos/) image?